docs/source/installation_instructions.rst

```diff
@@ -0,0 +1,91 @@
+.. The purpose of this file is to provide instructions on how to install the software necessary to access and run
+   the pytentiostat software package.
+
+Installation Instructions
+=========================
+
+.. seealso::
+   Instructions are listed below for installing Miniconda, but the full user documentation for Anaconda is available at
+   https://conda.io/projects/conda/en/latest/user-guide/install/index.html
+
+System Requirments
+------------------
+
+Before installing Anaconda, the following system requirements must be met:
+
+* 32- or 64-bit computer
+* 400 MB disk space
+* Windows, MacOS, or Linux
+* Python 2.7, 3.4, 3.5, 3.6, or 3.7
+* pycosat
+* PyYaml
+* Requests
+
+If you do not already have one of the required versions of Python, follow the instructions below.
+
+.. note::
+
+   We used Python 3.7 for the Pytentiostat software, so it is encouraged to download that version to ensure
+   compatibility.
+
+Python Installation
```

Comment:
This is not correct.  We want the python to be installed by miniconda.  Please delete.

Reply:
Got it, when I was going through the miniconda installation instructions I read something that said python was a system requirement. It seems like that is not the case.